requirements/core.txt

```diff
@@ -4,10 +4,11 @@
 # Without these, iris won't even import.
 
 cartopy
+cf_units>=1.2
```

Comment:
@bjlittle this needs to be `cf_units >=2`.

PS: I tested with a git install of latest `cf_units` and we still have some failures to fix here too.

Reply:
Yup, I've raised an issue on `cf_units`, see https://github.com/SciTools/cf_units/issues/97, I'm working on that now :wink:

Reply:
I guess that with your new tests this can be either 1 or 2, right?